spec/features/users/user_index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'capybara'
+
+RSpec.describe 'show all users', type: :feature do
+  let!(:users) do
+    User.all
+  end
+
+  context 'visit /users' do
+    before(:example) do
+      visit users_path
+    end
+
+    it 'I can see the profile picture for each user.' do
+      users.each do |user|
+        expect(page).to have_css("img[src*='#{user.photo}']")
+      end
+    end
+
+    it 'I can see the number of posts each user has written.' do
+      users.each do |user|
+        expect(page.find("#user_id_#{user.id}")).to have_content "Number of posts: #{user.posts_counter}"
+      end
+    end
+  end
+end
```

Comment:
- [ ] You should have the tests:
* that asserts you can see the username of all other users.
* that asserts you can see the profile picture for each user.
* that asserts you can see the number of posts each user has written.
* that redirects to the user's show page when you click on a user.

- [ ]  Please kindly add all these tests to your file, those tests are for the `User` index page 🙃